dags/openshift_nightlies/tasks/install/openshift.py

```diff
@@ -26,18 +25,21 @@ def __init__(self, dag, release: OpenshiftRelease):
         self.vars = var_loader.build_task_vars(
             release, task="install")
 
+        if 'airflow_executor_image' in self.vars.keys():
+            self.exec_config = var_loader.get_default_executor_config(self.vars['airflow_executor_image'])
+        else:
+            self.exec_config = var_loader.get_default_executor_config()
+
         # Airflow Variables
-        self.ansible_orchestrator = var_loader.get_secret(
-            "ansible_orchestrator", deserialize_json=True)
-
-        self.install_secrets = var_loader.get_secret(
-            f"openshift_install_config", deserialize_json=True)
-        self.aws_creds = var_loader.get_secret("aws_creds", deserialize_json=True)
-        self.gcp_creds = var_loader.get_secret("gcp_creds", deserialize_json=True)
-        self.azure_creds = var_loader.get_secret("azure_creds", deserialize_json=True)
-        self.ocp_pull_secret = var_loader.get_secret("osp_ocp_pull_creds")
-        self.openstack_creds = var_loader.get_secret("openstack_creds", deserialize_json=True)
-        self.release_stream_base_url = var_loader.get_secret("release_stream_base_url")
+        self.ansible_orchestrator = Variable.get("ansible_orchestrator", deserialize_json=True)
```

Comment:
every Variable we grab needs to be grabbed from `var_loader.get_secret`. It takes the same arguments

Reply:
ok, I ate this commit on the PR, needs to rebase